content/departments/engineering/teams/release/onboarding.md

```diff
@@ -30,7 +30,7 @@ Welcome to the Delivery team! This document will guide you through delivery-spec
   - #dev-ops
   - #dev-chat
   - #dev-accounce
-- Make sure you are added to the [Delivery](https://github.com/orgs/sourcegraph/teams/delivery) team in Sourcegraph GitHub org. Reach out to [#delivery-internal] if you're not.
+- Make sure you are added to the [Delivery](https://github.com/orgs/sourcegraph/teams/release) team in Sourcegraph GitHub org. Reach out to [#delivery-internal] if you're not.
```

Comment:
```suggestion
- Make sure you are added to the [Release](https://github.com/orgs/sourcegraph/teams/release) team in Sourcegraph GitHub org. Reach out to [#team-releases] if you're not.
```

Note: the Slack channel is called `#team-releases`, but I see references to `Release` here. I think the plural form is preferred?

Reply:
I've seen `Release` everywhere but the Slack channel naming (Miro chart, RFCs, etc...). Either sounds right to me though 🤷